web/.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:react-hooks/recommended',
     'plugin:@next/next/recommended',
+    'prettier',
```

Comment:
Nice miscellaneous change! Could just be a separate commit to main.

Reply:
actually this doesn't work :(